README.md

```diff
@@ -283,6 +188,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As suggested by the previous reviewer you did awesome work by giving the correct file name for the license. But please always make sure that your link is correct by cross-checking, as I can see that you forget to add the proper name with an extension which again gives a 404-page error. Kindly follow the suggestion to fix it quickly.

```suggestion
This project is [MIT](./MIT.md) licensed.
```

**Note:** *Please don't forget to customize your license with your name an information*